### PR TITLE
Created new branch for TC_no-42153 

### PR DESCRIPTION
**Pull request by Rohany7**

### CODE REVIEW

File: src/javalogical/LargestNum.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        }
+        return a[total-1];
+    }
+    public static void main(String args[]){
+        int a[]={1,2,5,6,3,2};
+        int b[]={44,66,99,77,33,22,55};
+        System.out.println("Largest: "+getLargest(a,6));
+        System.out.println("Largest: "+getLargest(b,7));
+    }
+}
```
Comment (rohanyemul) [inline src/javalogical/LargestNum.java:26]: Looks good

Comment (Rohany7) [reply src/javalogical/LargestNum.java:26]: thanks, rony!

Comment (rohanyemul) [reply src/javalogical/LargestNum.java:26]: Create a new class for to find smallest number!

Comment (Rohany7) [reply src/javalogical/LargestNum.java:26]: Ok Sure...

File: src/javalogical/SmallestNum.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        }
+        return a[0];
+    }
+    public static void main(String args[]){
+        int a[]={1,2,5,6,3,2};
+        int b[]={44,66,99,77,33,22,55};
+        System.out.println("Smallest: "+getSmallest(a,6));
+        System.out.println("Smallest: "+getSmallest(b,7));
+    }
+}
```
Comment (rohanyemul) [inline src/javalogical/SmallestNum.java:26]: So cool good job man!

**Review by rohanyemul — CHANGES_REQUESTED** [review]

File: src/javalogical/SmallestNum.java (hunk 2 above)
Comment (Rohany7) [reply src/javalogical/SmallestNum.java:26]: thanks, rony!

**Pull request merged**